sources/install.sh

```diff
@@ -1192,6 +1192,13 @@ function ghidra() {
   rm /tmp/ghidra_10.1.2_PUBLIC_20220125.zip
 }
 
+function ida() {
```

Comment:
Could you rename that function to `install_ida` please ?

Reply:
Yes no worries, it's done ;)